src/foam/u2/view/UnstyledTableView.js

```diff
@@ -276,47 +274,14 @@ foam.CLASS({
                       }).
                     end();
                 }).
-                call(function() {
-                  var modelActions = view.of.getAxiomsByClass(foam.core.Action);
-                  var actions = Array.isArray(view.contextMenuActions) ?
-                    view.contextMenuActions.concat(modelActions) :
-                    modelActions;
-                  var overlay = view.OverlayDropdown.create();
-                  return this.start('td').
-                    callIf(actions.length > 0, function() {
-                      overlay.forEach(actions, function(action) {
-                        this.
-                          start().
-                            show(action.createIsAvailable$(view.ConstantSlot.create({ value: obj }))).
-                            addClass(view.myClass('context-menu-item')).
-                            add(action.label).
-                            call(async function() {
-                              if ( await action.isEnabledFor(obj) ) {
-                                this.on('click', function(evt) {
-                                  action.maybeCall(view.__subContext__, obj);
-                                });
-                              } else {
-                                this.addClass('disabled');
-                              }
-                            }).
-                          end();
-                      });
-                      view.ctrl.add(overlay);
-                    }).
-                    style({ 'text-align': 'right' }).
-                    start('span').
-                      addClass(view.myClass('vertDots')).
-                      addClass(view.myClass('noselect')).
-                      enableClass('disabled', actions.length === 0).
-                      callIf(actions.length > 0, function() {
-                        this.on('click', function(evt) {
-                          overlay.open(evt.clientX, evt.clientY);
-                        });
-                      }).
-                      add(view.vertMenuIcon).
-                    end().
-                  end();
-                });
+                start('td').
+                  addClass(view.myClass('context-menu-cell')).
+                  tag({
```

Comment:
The style I would like to move to for `tag(` and `start(` calls like this is to add
```
requires: [
  'foam.u2.view.OverlayActionListView'
]
```

and do

```
.tag(this.OverlayActionListView, { data: actions, obj: obj })
```

This will make the build tool's job easier.